### DISCUSSION TITLE
Newbie questions

### DISCUSSION THREAD

**Discussion by tlamoureux24** [Louisvdw/dbus-serialbattery#1080] [Q&A] [answered]
Hello,
I have just two little questions please : i have a seplos V3.0 currently link to my Cerbo GX using CAN protocol.
It works but lot's of data is missing.
If i install and setup serialbattery driver, should i have more data available in cerbo ?
And if i install driver, i suppose i have to remove CAN link and use instead RS485, right ?
Thank you and have a good day !!

**Comment by mr-manuel** [comment, marked as answer]
Check the documentation: https://mr-manuel.github.io/venus-os_dbus-serialbattery_docs/general/features#bms-feature-comparison

Yes you have to use RS485.
